common/rpc/interceptor/namespace_validator.go

```diff
@@ -304,6 +304,9 @@ func (ni *NamespaceValidatorInterceptor) extractNamespaceFromRequest(req interfa
 			return ni.namespaceRegistry.GetNamespaceByID(namespaceID)
 		}
 		return ni.namespaceRegistry.GetNamespace(namespaceName)
+	case *adminservice.DescribeHistoryHostRequest:
```

Comment:
Why not just add to to the next case?
If namespace is empty (and it is for this case) - it will be the same.

Reply:
Because next case is all about search attributes.